pkgs/tools/filesystems/btrfs-snap/default.nix

```diff
@@ -0,0 +1,27 @@
+{ bash, btrfs-progs, fetchFromGitHub, makeWrapper, lib, stdenv }:
+stdenv.mkDerivation rec {
+  pname = "btrfs-snap";
+  version = "1.7.3";
+  src = fetchFromGitHub {
+    owner = "jf647";
+    repo = pname;
+    rev = "${version}";
+    sha256 = "sha256-SDzLjgNRuR9XpmcYCD9T10MLS+//+pWFGDiTAb8NiLQ=";
+  };
+  buildInputs = [ bash btrfs-progs ];
+  nativeBuildInputs = [ makeWrapper ];
+  dontConfigure = true;
+  dontBuild = true;
+  installPhase = ''
+    mkdir -p $out/bin
+    cp * $out/bin/
+    wrapProgram $out/bin/btrfs-snap --prefix PATH : ${lib.makeBinPath [ btrfs-progs ]}
+  '';
+  meta = with lib; {
+    description = "btrfs-snap creates and maintains the history of snapshots of btrfs filesystems.";
+    homepage = "https://github.com/jf647/btrfs-snap";
+    license = licenses.gpl3;
```

Comment:
gpl3Only or gpl3Plus ? Please precise, and if necessary, ask upstream to clarify.

Reply:
TIL `gpl3Only`. Fixed.